emscripten/luna-utils.js

```diff
@@ -0,0 +1,44 @@
+var Module = { 'preRun': function() {
+
+callLuna = function(luaFileName, tnsFileName)
+{
+   Module.callMain([luaFileName, tnsFileName]);
```

Comment:
Then maybe just run Module["main"] directly? The return code is definitely useful outside of luna